modules/context/context.go

```diff
@@ -174,13 +175,19 @@ func (ctx *Context) HasValue(name string) bool {
 	return ok
 }
 
+var precedingSlashesRE = regexp.MustCompile(`^/[\\/]+`)
```

Comment:
Shouldn't forward slash be escaped in typical regex? Also, is there any need to check for multiple repetitions?
```suggestion
var precedingSlashesRE = regexp.MustCompile(`^\/\\\/`)
```

Reply:
No the forward slash doesn't need to be escaped. 

You're right though that all we actually need is `[0]=/ || \ AND [1] = / || \` so we can probably get rid of the regexp